### DISCUSSION TITLE
Is CD changer feature actually a thing on BlueSCSI v1?

### DISCUSSION THREAD

**Discussion by MrGasS** [erichelgeson/BlueSCSI-v1#258] [Q&A] [answered]
Just asking because file transferring feature works fine, but CD changer's application says that it can't see a compatible BlueSCSI.

I'm using my custom PCB (blueSCSI evolved) with a legit BluePill on my beloved IIsi, it's running System 7.5.3 in italian.

Thank you :)

**Comment by erichelgeson** [comment, marked as answer]
Correct - there is no CD switching on the v1 - while it could be done it would take some effort to port over. I thought I had this in the readme but will double check and add it to the wiki as well.

**Reply by MrGasS** [reply]
Thank you a lot
